concordium-std-derive/Cargo.toml

```diff
@@ -17,7 +17,7 @@ proc-macro2 = "1.0"
 
```

Comment:
We also want to bump the version of concordium-std-derive. And add to its changelog that we now support `event` in the schema derivation macros.